tests/test_secret_manager.py

```diff
@@ -0,0 +1,12 @@
+import responses
+
+from wayscript import secret_manager as ws_secret_manager
+
+
+@responses.activate
+def test_set_secret(patch_client_get_url):
+    """Test set secret"""
+    responses.add(responses.POST, patch_client_get_url, json={}, status=200)
+
+    callable = getattr(ws_secret_manager, 'set_secret')
+    assert callable("test_key", "test_val") == True
```

Comment:
What's the difference between this and
```
result = ws_secret_manager.set_secret('test_key', 'test_val')
assert result == True
```

Reply:
Probably identical, I was following the pattern set by the rest of the SDK's tests.

Reply:
Feel free to keep it but I don't think this a pattern we want continue using.  The best example I found of this was in [test_context.py](https://github.com/wayscript/wayscript-python/blob/55abcf4f22869fccf6dbc7853444b70339683f2a/tests/test_context.py) which appears do it both ways.